content/app/guides/multi-app-solution/instructions/_index.en.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Instructions for making a multi-app solution in Altinn
+linktitle: Instructions
+description: Explanations of how to go about when creating a general multi-app solution
+weight: 20
+aliases:
+
+- /app/multi-app-solution/instructions/
+
+---
+
+## Prerequisites
+
+Before continue reading the technical implementation details, please make sure the
+required prerequisites are met.
+
+### Functional Prerequisites
+
+1. The app-developer must have access to create, develop and deploy applications owned by an organization.
+2. Existing basic Altinn form where the content of the data model is assessed against the intended data flow between
```

Comment:
This point was a bit complex. Is the main point to have Application A already up and running or that they should have though about which data points they want to forward? 

Could we consider: 
An existing Altinn form (application A) of which identified parts of the data is intended to be forwarded to application B.

Reply:
The intention of the point was to emphasie that they have given some thought to the data flow, yes.

Maybe an even clearer formulation could be:
An existing Altinn form (application A) where the data that is intended forwarded to application B is identified.

Reply:
Sounds good. Go for it :)